### PR TITLE
blog week 32

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Add blog-post for week 32&33.
- [x] The content will be regenerated once approved.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
Anything missing? There were a lot of refactoring/deployment changes that I found not interesting for adding to the blog.

File: content/posts/August-2020.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "August 2020"
+date: 2020-08-13
+draft: false
+---
+
+## Week 32 (August 3rd - August 7th) and 33 (August 10th - August 13th)
+
+- Last week, we renamed our GitHub organisation to `packit`.
```
Comment (jpopelka) [inline content/posts/August-2020.md:9]: 'Last week' is relative
```suggestion
-We renamed our GitHub organisation to `packit`.
```

File: content/posts/August-2020.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  That user was inactive for a long time and [Hunor]'s request was successful.
```
Comment (jpopelka) [inline content/posts/August-2020.md:10]:
```suggestion
  That user had been inactive for a long time and [Hunor]'s request was successful.
```

File: content/posts/August-2020.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/posts/August-2020.md:9]: Sorry for editing your comment to contain the space.

File: content/posts/August-2020.md (hunk 2 above)
Comment (lachmanfrantisek) [reply content/posts/August-2020.md:10]: Thanks! :facepalm:

**Pull request merged**